aligned_accessor/aligned_accessor.md

```diff
@@ -0,0 +1,294 @@
+  auto y_alloc = allocate_raw<float, max_byte_alignment>(num_elements);
+
+  aligned_mdspan<max_byte_alignment> x(x_alloc.get());
+  aligned_mdspan<max_byte_alignment> y(y_alloc.get());
+
+  fill_x(x); // automatic conversion from 32-byte aligned to 16-byte aligned
+  fill_y(y); // automatic conversion again
+
+  vectorized_axpy(y, alpha, x);
+  return vectorized_norm(y);
+}
+```
+
+# Wording
+
+> Text in blockquotes is not proposed wording, but rather instructions for generating proposed wording.
+> The � character is used to denote a placeholder section number which the editor shall determine.
+>
+> In *[version.syn]*, add
+
+```c++
+#define __cpp_lib_aligned_accessor YYYYMML // also in <mdspan>
+```
+
+> Adjust the placeholder value as needed so as to denote this proposal's date of adoption.
+>
+> To the Header `<mdspan>` synopsis **[mdspan.syn]**, after `class default_accessor` and before `class mdspan`, add the following.
+
+```c++
+// [mdspan.accessor.aligned], class template aligned_accessor
+template<class ElementType, size_t byte_alignment>
+  class aligned_accessor;
+```
+
+> At the end of **[mdspan.accessor.default]** and before **[mdspan.mdspan]**, add all the material that follows.
+
+## Add subsection � [mdspan.accessor.aligned] with the following
+
+<b> � Class template `aligned_accessor` [mdspan.accessor.aligned] </b>
+
+<b> �.1 Overview [mdspan.accessor.aligned.overview] </b>
+
+```c++
+template<class ElementType, size_t the_byte_alignment>
+struct aligned_accessor {
+  using offset_policy = default_accessor<ElementType>;
+  using element_type = ElementType;
+  using reference = ElementType&;
+  using data_handle_type = ElementType*;
+
+  static constexpr size_t byte_alignment = the_byte_alignment;
+
+  constexpr aligned_accessor() noexcept = default;
+
+  template<class OtherElementType, size_t other_byte_alignment>
+    constexpr aligned_accessor(
+      aligned_accessor<OtherElementType, other_byte_alignment>) noexcept;
+
+  constexpr operator default_accessor<element_type>() const {
+    return {};
+  }
+
+  constexpr reference access(data_handle_type p, size_t i) const noexcept;
+
+  constexpr typename offset_policy::data_handle_type
+    offset(data_handle_type p, size_t i) const noexcept;
+
+  constexpr static bool is_sufficiently_aligned(data_handle_type p);
+};
+```
+
+[1]{.pnum} *Mandates*: `byte_alignment` is a power of two.
+
+[2]{.pnum} `aligned_accessor` meets the accessor policy requirements.
+
+[3]{.pnum} `ElementType`  is required to be a complete object type that is neither an abstract class type nor an array type.
+
+[4]{.pnum} Each specialization of `aligned_accessor` is a trivially copyable type that models `semiregular`.
+
+[5]{.pnum} $[0, n)$ is an accessible range for an object `p` of type `data_handle_type` and an object of type `aligned_accessor` if and only if $[$`p`, `p` + $n)$ is a valid range.
+
+## Members [mdspan.accessor.aligned.members]
+
```

Comment:
Default constructor is missing

Reply:
Thanks for noticing this, though I think it's OK.  ` constexpr aligned_accessor() noexcept = default;`  is in the class overview.  Also, the wording is consistent with `default_accessor`, which doesn't have a separate entry for the defaulted default constructor: 
https://eel.is/c++draft/views.multidim#mdspan.accessor.default .

Reply:
You are right.  Sorry about the noise :)